packages/manager/modules/vps/src/dashboard/migrate/migrate.html

```diff
@@ -0,0 +1,115 @@
+<h4 data-translate="vps_dashboard_migrate_title"></h4>
+<div class="w-25 mt-3 oui-select-picker">
+    <label class="oui-select-picker__label-container">
+        <span class="oui-select-picker__body">
+            <h6 data-translate="vps_dashboard_migrate_current_range_title"></h6>
+            <div class="oui-select-picker__section">
+                <dl class="vps-dl-horizontal">
+                    <dt
+                        class="vps-item-horizontal text-bold"
+                        data-translate="vps_dashboard_migrate_configuration_label_core"
+                    ></dt>
+                    <dd
+                        class="vps-item-horizontal"
+                        data-translate="vps_dashboard_migrate_configuration_vcore"
+                        data-translate-values="{ count: $ctrl.stateVps.vcore }"
+                    ></dd>
+                    <dt
+                        class="vps-item-horizontal text-bold"
+                        data-translate="vps_dashboard_migrate_configuration_label_ram"
+                    ></dt>
+                    <dd
+                        class="vps-item-horizontal"
+                        data-ng-bind=":: $ctrl.formatMemory($ctrl.stateVps.model.memory) | cucBytes:0:false:'GB'"
+                    ></dd>
+                    <dt
+                        class="vps-item-horizontal text-bold"
+                        data-translate="vps_dashboard_migrate_configuration_label_disk"
+                    ></dt>
+                    <dd
+                        class="vps-item-horizontal"
+                        data-ng-bind=":: $ctrl.stateVps.model.disk | cucBytes:0:false:'GB'"
+                    ></dd>
+                </dl>
+            </div>
+            <div class="oui-select-picker__footer">
+                <strong
+                    data-ng-bind="$ctrl.currentPrice.priceWithoutTax.text"
+                ></strong>
+                /<span
+                    data-translate="vps_dashboard_migrate_price_per_month"
+                ></span>
+            </div>
+        </span>
+    </label>
+</div>
+
+<p class="mt-3">
+    <span data-translate="vps_dashboard_migrate_information_1"></span><br />
+    <span data-translate="vps_dashboard_migrate_information_2"></span>
+</p>
+
+<p class="mt-3">
+    <b data-translate="vps_dashboard_migrate_link_information"></b>
```

Comment:
Out of curiosity, is there any reason to not prefer using `strong` element?

Reply:
No reason :) ill switch to `<strong>`